README.md

```diff
@@ -2,37 +2,39 @@
 
 ## About this task
 
-Think of this as an open source project. How would this have to look, in order for you to be impressed with it if you were to find it on Github? Now go do that.
+Think of this as an open source project. How would this have to look in order for you to be impressed with it, if you were to find it on GitHub? Now go and do that.
 
-Try to limit the amount of time you spend on this to 90-120 minutes. However, feel free to spend more - just make sure you're happy with your submission!
+Try to limit the amount of time you spend on this to 90-120 minutes. Feel free to spend more time - make sure you are happy with your submission!
```

Comment:
Without the `however`, these sentences are now at odds with either other. `try to limit` vs `feel free to spend more`. Which is it?

Personally, I would also amend this to `Please spend at least 90 minutes on this test. Feel free to take more time if you wish - make sure you are happy with your submission!`

Reply:
Updated to your recommendation.